src/main/scala/cromwell/Main.scala

```diff
@@ -66,6 +67,10 @@ object Main extends App {
     argCapitalized = arg.capitalize
     action <- Actions.values find (_.toString == argCapitalized)
   } yield action
+
+  def deprecationNotice(): Unit = {
+    Console.err.println("This functionality is deprecated. Please use wdltool: https://github.com/broadinstitute/wdltool")
```

Comment:
This is fine as long as github is the main way people will download cromwell.

As soon as people start getting Cromwell in binary form (e.g. homebrew, website, ...?) this should point to our official release place.